src/imap_mag/main.py

```diff
@@ -181,7 +183,11 @@ def fetch_binary(
     packet: str = appUtils.getPacketFromApID(apid)
     logging.info(f"Downloading raw packet {packet} from {start_date} to {end_date}.")
 
-    poda = webPODA.WebPODA(auth_code, configFile.work_folder)
+    poda = WebPODA(
+        auth_code,
+        configFile.work_folder,
+        configFile.api.webpoda_url if configFile.api else None,
```

Comment:
this should crash/error if the url is none and not pass in None

Reply:
The url is `None` by default. If it's `None` we use the "real" address.